ballonsballons-at/ballons-jaen
# feat: add full parallax hero

## Description

This pull request adds the following features to the application:

- Background mobile hero
- Background animation for faster transition
- Scroll down hint

These features will improve the user experience of the application and make it more visually appealing.

## Changes Made

I have implemented these features in my branch and thoroughly tested them to ensure they function as expected. Here is a brief description of each feature:

- The background mobile hero provides an eye-catching image that will leave a lasting impression on users. This feature also helps to establish the brand identity of the application.
- The background animation for faster transition creates a smoother and more seamless user experience. This feature helps to reduce the load time of the application, making it more user-friendly.
- The scroll down hint provides a clear indication to the users that there is more content to be explored, making the application more intuitive.

## How to Test

To test these features, simply pull this branch and run the application. You should see the new features implemented.

## Checklist

- [x] Follows code style and formatting guidelines
